Monika After Story/game/event-handler.rpy

```diff
@@ -2467,8 +2469,8 @@ label mas_bookmarks:
         bookmarkedlist = mas_get_player_bookmarks()
 
         bookmarkedlist.sort()
-        remove_bookmark = (mas_getEV('mas_topic_unbookmark').prompt, mas_getEV('mas_topic_unbookmark').eventlabel, False, False, 20)
-        return_prompt_back = ("Nevermind.", False, False, False, 0)
+        remove_bookmark = (mas_getEV('mas_topic_unbookmark').prompt, 'mas_topic_unbookmark', False, False, 20)
+        return_prompt_back = ("Nevermind.", "prompt", False, False, 0)
```

Comment:
hold up, line 2480 would eval to true now since the return value contains something. False works fine if you change that if statement a little like:
```python
if topic_choice:
    # same stuff
    return True

return False
```